### PR TITLE
CB-1584 Azure credential run into an error during the converting

### PR DESCRIPTION
**Pull request by topolyai5**

### CODE REVIEW

File: cloud-aws/src/main/resources/definitions/aws-credential.json [hunk 1]
```diff
@@ -17,6 +17,24 @@
       "type": "String",
       "sensitive": false,
       "optional": true
+    },
+    {
+      "name": "keyBased.accessKey",
```
Comment (keyki) [inline cloud-aws/src/main/resources/definitions/aws-credential.json]: is this needed?

File: cloud-azure/src/main/resources/definitions/azure-credential.json [hunk 2]
```diff
@@ -57,6 +59,64 @@
...
+      "sensitive": false
+    },
+    {
+      "name": "secretKey",
+      "type": "String",
+      "optional": true,
+      "sensitive": true
+    },
+    {
+      "name": "accessKey",
```
Comment (keyki) [inline cloud-azure/src/main/resources/definitions/azure-credential.json]: duplicate

File: cloud-azure/src/main/resources/definitions/azure-credential.json [hunk 3]
```diff
@@ -57,6 +59,58 @@
...
       "optional": true,
       "sensitive": false
+    },
+    {
+      "name": "subscriptionId",
+      "type": "String",
+      "sensitive": false
+    },
+    {
+      "name": "tenantId",
```
Comment (keyki) [inline cloud-azure/src/main/resources/definitions/azure-credential.json]: should these be optional? new credentials won't have these fields.

Comment (topolyai5) [reply cloud-azure/src/main/resources/definitions/azure-credential.json]: it is the old one, the new one is optional

**Pull request merged**